audio/sfx/unused_cry_1.asm

```diff
@@ -8,8 +9,7 @@ SFX_Unused_1_Ch5:
 	square_note 8, 7, 1, 1924
 	sound_ret
 
-
-SFX_Unused_1_Ch6:
```

Comment:
Can you restore the double blank line between channels? This is the separation used in all sfx/song files.

Reply:
Fixed!